presentation/src/main/java/com/anytypeio/anytype/presentation/home/HomeScreenViewModel.kt

```diff
@@ -434,7 +432,11 @@ class HomeScreenViewModel(
         widgets: List<WidgetView>
     ): List<WidgetView> {
         return buildList {
-            addAll(widgets)
+            addAll(
```

Comment:
val filteredWidgets = widgets.filterNot { view ->
        view is WidgetView.Bin && (view.isEmpty || view.isLoading)
    }
addAll(filteredWidgets)

Reply:
fixed